README.md

```diff
@@ -1,2 +1,137 @@
-# math-magician
-In this project I am going to create a simple react app
+<a name="readme-top"></a>
+# React App
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#In this project we build simple Book list application that can add and remove the Books)
+  - [🛠 Built With](#build-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [React-App] <a name="about-project"></a>
+
+> In this project we build simple react App that is diplaying logo and added some animation using css.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://reactjs.org/">Javascript</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+  </ul>
+</details>
+<ul>
+    <li><a href="https://reactjs.org/">React</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[DOM-manipulating]**
+- **[Content-loaded-quickly-on-UI]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Browser
+- git
+- code editor
+
+### Setup
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone the repository using:
+```
+   https://github.com/Shaheryar0054/math-magician.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] It would be nice if you add the following commands in the Getting Started section which are required to start the application 👍

````
### Install
```
npm install
```
### Usage
```
npm start
```
```